### DISCUSSION TITLE
Creating Material UI versions of the components

### DISCUSSION THREAD

**Discussion by Folcon** [day8/re-com#273] [Show and tell]
This might not be of interest, however I've started creating Material UI versions of some of the components as [defined here](https://material-ui.com/demos/tabs/).

I'm not sure if you want to ignore this, or there might be some interest in merging the components?

I'm doing it on an as-needed basis, and the code is available here:
https://github.com/Folcon/re-com/

**Comment by superstructor** [comment]
@Folcon Thanks for sharing. We have some exciting changes coming soon for styling/theming, which should enable such variations to be shipped as a layer on top of re-com, such as a library, without re-implementing the components entirely.

**Reply by Folcon** [reply]
@superstructor: That's great to hear, looking forward to getting additional info!

**Reply by superstructor** [reply]
@Folcon The plan that we are beginning to implement is at https://github.com/day8/re-com/issues/284
